server/app/services/feed-object-service.js

```diff
@@ -151,17 +152,12 @@ async function updateFeedObject(feedObjectId, updatedFeedObject, _embedded) {
   try {
     if (updatedFeedObject.medias) {
       const oldMedias = await currentFeedObject.getMedias();
-      // eslint-disable-next-line
-      for (const oldMedia of oldMedias) {
-        // eslint-disable-next-line
-        await oldMedia.destroy();
-      }
-      // eslint-disable-next-line
-      for (const newMedia of updatedFeedObject.medias) {
-        // eslint-disable-next-line
-        await newMedia.save();
-      }
-      await currentFeedObject.setMedias(updatedFeedObject.medias, { transaction });
+      await Promise.all(oldMedias.map(m => m.destroy({ transaction })));
+      const newMediasCreatePromises = updatedFeedObject.medias.map(m => m.save({ transaction }));
+      await Promise.all(newMediasCreatePromises)
+        .then(async (newMedias) => {
+          await currentFeedObject.setMedias(newMedias, { transaction });
+        });
```

Comment:
Why so much requests? We could delete every old medias in one request, then saving every updated media at once with a bulkCreate and then setting relation with a setMedias(array)?

Reply:
Of course :) I've never heard of bulk in sequelize !